src/client/pages/Varen/useVarenListPage.hook.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  varenMeerInformatieLink,
+  tableConfig,
+  exploitatieVergunningAanvragen,
+} from './Varen-thema-config';
+import { VarenVergunningFrontend } from '../../../server/services/varen/config-and-types';
+import { isError, isLoading } from '../../../universal/helpers/api';
+import { addLinkElementToProperty } from '../../components/Table/TableV2';
+import { useAppStateGetter } from '../../hooks/useAppState';
+
+export type VarenVergunningKind = 'open' | 'afgehandeld' | 'overgedragen';
+
+export function useVarenListPage(kind: VarenVergunningKind) {
```

Comment:
Is dit niet min-of-meer een duplicatie van de useVarenThemaData hook? Die kun je ook gewoon gebruiken in de listpage.